addOns/soap/src/main/java/org/zaproxy/zap/extension/soap/CommonScanRuleInfo.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.soap;
+
+public interface CommonScanRuleInfo {
+
+    default String getHelpLink() {
+        return "https://www.zaproxy.org/docs/desktop/addons/soap-support/#scanning";
```

Comment:
Why not link to exact scan rule?

Reply:
To the row in the table? Or are you picturing it displayed differently?

Reply:
Row would be fine but maybe we should use the same format everywhere (I know GraphQL is using the same).

Reply:
Okay, I made a separate Alerts page like GraphQL, and linked the rows.